scripts/xonsh

```diff
@@ -1,3 +1,3 @@
-#!/usr/bin/env python -u
-from xonsh.main import main
-main()
+#!/usr/bin/env sh
+export PYTHONUNBUFFERED=1
+/usr/bin/env python -c "from xonsh.main import main; main()"
```

Comment:
Is there a reason to do this instead of `/usr/bin/env python -m xonsh.main`?

Not sure whether there is any practical difference between the two.